meta-mender-core/recipes-mender/mender-client/mender_git.bb

```diff
@@ -0,0 +1,14 @@
+require mender-client-cpp.inc
+
+LICENSE = "Apache-2.0 & BSD-3-Clause"
+
+LIC_FILES_CHKSUM = " \
+    file://LICENSE;md5=4cd0c347af5bce5ccf3b3d5439a2ea87 \
+    file://vendor/json/LICENSE.MIT;md5=f969127d7b7ed0a8a63c2bbeae002588 \
+    file://vendor/json/docs/mkdocs/docs/home/license.md;md5=970ea048f6ea7d04eeb3ba3ef9ebca40 \
+    file://vendor/lmdbxx/UNLICENSE;md5=7246f848faa4e9c9fc0ea91122d6e680 \
+"
+
+SRCREV = "${@mender_autorev_if_git_version(d)}"
+
+SRC_URI = "gitsm://github.com/oleorhagen/mender;protocol=https;branch=cpp-cmake-files"
```

Comment:
Will fix once the this has passed CI alongside the mender-client PR: https://github.com/mendersoftware/mender/pull/1112